lib/plugins/aws/deploy/lib/initializeResources.js

```diff
@@ -1,63 +1,27 @@
 'use strict';
 
-const _ = require('lodash');
 const path = require('path');
 const BbPromise = require('bluebird');
 
 module.exports = {
   initializeResources() {
-    const coreCFTemplate = this.serverless.utils.readFileSync(
+    // save the custom resources (e.g. Resources, Outputs etc.) the user has added
+    // so that they can be merged in later on
+    this.serverless.service.customProviderResources = {};
+
+    if (this.serverless.service.resources && this.serverless.service.resources.Resources) {
+      this.serverless.service.customProviderResources = this.serverless.service.resources;
```

Comment:
I think it would be better to not have another place to do custom resources. Otherwise plugins might write additional resources there and they don't get copied or copied at the wrong time.

Imho loading the core-cf file, deploying it to Cloudformation and then adding it to service.resources should all be done in the createStack.js file. If we try to do it here we have to do some workarounds that could lead to other issues, when we do it all in createStack.js we don't have any issues really